README.md

```diff
@@ -10,9 +10,12 @@ php-coveralls
 
 PHP client library for [Coveralls](https://coveralls.io). 
 
-# API doc
+# Future deprecations
 
-[API doc](http://satooshi.github.io/php-coveralls/api/) is generated by [ApiGen](http://apigen.org/)
+We will make a breaking change regarding to the config file format on
```

Comment:
1. write about it when you done that changes.
2. write about it in changelog, not readme file